components/articles/cards/ArticleCard.js

```diff
@@ -78,32 +82,36 @@ const ArticleCard = props => {
     type,
     slug,
     tagsByTopic,
+    onBackground,
   } = props;
   const { brands = [], authors = [] } = tagsByTopic;
   const { short, full } = ACTION_BY_TYPE[type];
 
   return (
     <CardWrapper
       size={size}
+      sizeClass={styles[`card-size-${size}`]}
+      blockContextClass={blockContext.map(ctx => styles[`block-${ctx}`]).join(' ')}
```

Comment:
used only once? mb extract to helper?

Reply:
used twice
well I don't know it looks too simple and short to implement a helper here